src/shared/types/abTests/epic.ts

```diff
@@ -112,8 +112,8 @@ export interface EpicTest extends EpicTestFromTool {
 export const epicTestFromToolSchema = testSchema.extend({
     name: z.string(),
     status: testStatusSchema,
-    locations: z.array(countryGroupIdSchema),
-    targetedCountries: z.array(z.string()).optional(),
+    locations: z.array(countryGroupIdSchema), //deprecated
+    targetedCountries: targetedCountriesSchema.optional(),
```

Comment:
since it's countryGroups as well as countries, would something like regionTargeting be clearer?

Reply:
For the field name or the schema? Or both? I'm learning towards changing them both